web-ui/src/main/resources/catalog/templates/admin/settings/static-pages.html

```diff
@@ -306,15 +306,41 @@
                 class="form-control"
                 required=""
                 data-ng-model="staticPageSelected.status"
+                data-ng-click="updateGroupSelection()"
               >
                 <option value="HIDDEN">{{'staticPageStatus-HIDDEN' | translate }}</option>
                 <option value="PRIVATE">
                   {{'staticPageStatus-PRIVATE' | translate }}
                 </option>
+                <option value="PRIVATE_GROUP">
+                  {{'staticPageStatus-PRIVATE_GROUP' | translate }}
+                </option>
                 <option value="PUBLIC">{{'staticPageStatus-PUBLIC' | translate }}</option>
               </select>
             </div>
           </div>
+
+          <div class="form-group">
+            <label class="control-label col-sm-3" data-translate="">group</label>
+
+            <div class="col-sm-9">
+              <select
+                name="group"
+                class="form-control"
+                data-ng-disabled="!isGroupEnabled"
+                data-ng-readonly="!isGroupEnabled"
+                data-ng-model="staticPageSelected.group"
+              >
+                <option
+                  data-ng-repeat="g in groups"
+                  value="{{g.name}}"
+                  data-ng-selected="g.name==='dfo'"
```

Comment:
Could you explain this?
`g.name==='dfo'`

Reply:
opps. It was my localhost testing code which I hardcoded. Should change to something dynamically